Framework/CurveFitting/src/Functions/CubicSpline.cpp

```diff
@@ -79,26 +79,39 @@ void CubicSpline::setupInput(boost::scoped_array<double> &x,
   bool xSortFlag = false;
```

Comment:
I would change this variable name, I found it to be misleading. I was expecting it to be true if the data was sorted correctly.